lib/Model/Summary.php

```diff
@@ -26,6 +26,8 @@ class Summary
     private int $nbFailures = 0;
     private int $nbAssertions = 0;
     private int $nbErrors = 0;
+
+    /** @var array<string, array<int|float>> */
```

Comment:
_could_ use an array shape here, although maybe better as a type alias if it's referenced again

Reply:
Let me think a bit.